docs/.vuepress/sidebar/en.js

```diff
@@ -616,6 +615,7 @@ module.exports = [
               "SHOW CREATE FUNCTION",
               "SHOW CREATE ROUTINE LOAD",
               "SHOW DATA",
+              "SHOW DATA SKEW",
```

Comment:
This name must be same as the name of document ~

Reply:
done